Article/Article.js

```diff
@@ -2,25 +2,41 @@
 
 class Article {
   constructor(domElement) {
-    // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.domElement = domElement;
+    this.expandButton = domElement.querySelector('.expandButton');
+    this.expandButton.innerText = 'expand';
+    this.expandButton.addEventListener('click', item => this.expandArticle(item));
+    // this.closeButton = domElement.querySelector('.article-close');
+    // this.closeButton.addEventListener('click', item => this.closeButton())
   }
-
   expandArticle() {
-    // Using our reference to the domElement, toggle a class to expand or hide the article.
+      this.domElement.classList.toggle("article-open");
+      TweenMax.fromTo(this.domElement, 3, 
```

Comment:
Here is where an if statement should be that check if `article-open` is already in the classList, if it is, don't launch the opening animation instead launch the closing animation